content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,213 @@
+    type="video/mp4"
+  />
+</video>
+
+### Instalación
+
+Primero, necesitaremos instalar todas las dependencias necesarias.
+
+```bash
+yarn add @storybook/addon-knobs
+```
+
+Registra Knobs en tu archivo `.storybook/addons.js`.
+
+```javascript
+// .storybook/addons.js
+
+import '@storybook/addon-actions/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-links/register';
+```
+
+<div class = "aside">
+<strong> 📝 ¡El orden de registro de complementos es importante! </strong>
+<br/>
+El orden en que enumere estos complementos determinará el orden en que aparecerán como pestañas en su panel de complementos (para aquellos que aparecen allí).
+</div>
+
+¡Eso es! Es hora de usarlo en una historia.
+
+### Uso
+
+Usemos el tipo de knob de objeto en el componente `Tarea`.
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+
+import { storiesOf } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de las historias de `Task`, pase`withKnobs` como parámetro a la función `addDecorator()`:
+
+```javascript
+// src/components/Task.stories.js
+
+storiesOf('Task', module)
+  .addDecorator(withKnobs)
+  .add(/*...*/);
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+storiesOf('Task', module)
+  .addDecorator(withKnobs)
+  .add('default', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      props: {
+        task: {
+          type: Object,
+          default: object('task', { ...task }),
+        },
+      },
+      methods,
+    };
+  })
+  .add('pinned', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      data: () => ({ task: { ...task, state: 'TASK_PINNED' } }),
+      methods,
+    };
+  })
+  .add('archived', () => {
+    return {
+      components: { Task },
+      template: `<task :task="task" @archiveTask="onArchiveTask" @pinTask="onPinTask"/>`,
+      data: () => ({ task: { ...task, state: 'TASK_ARCHIVED' } }),
+      methods,
+    };
+  });
+```
+
+Ahora debería aparecer una nueva pestaña "Knobs" al lado de la pestaña "Action Logger" en el panel inferior.
+
+Como se documenta [here](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), el tipo `object` del knob acepta una etiqueta y un objeto predeterminado como parámetros. La etiqueta es constante y se muestra a la izquierda de un campo de texto en el panel de complementos. El objeto que ha pasado se representará como un blob JSON editable. ¡Siempre que envíe un JSON válido, su componente se ajustará en función de los datos que se pasan al objeto!
```

Comment:
Change "here" to "aqui" if you don't mind, in order to avoid mixing the language usage.